### PR TITLE
Bump version to 1.2.2 and update CHANGELOG

### PR DESCRIPTION
**Pull request by systemcatch**
## Pull Request Checklist

Thank you for taking the time to improve Arrow! Before submitting your pull request, please check all *appropriate* boxes:

- [ ] 🧪  Added **tests** for changed code.
- [ ] 🛠️  All tests **pass** when run locally (run `tox` or `make test` to find out!).
- [ ] 🧹  All linting checks **pass** when run locally (run `tox -e lint` or `make lint` to find out!).
- [ ] 📚  Updated **documentation** for changed code.
- [ ] ⏩  Code is **up-to-date** with the `master` branch.

If you have *any* questions about your code changes or any of the points above, please submit your questions along with the pull request and we will try our best to help!

### CODE REVIEW

**Review by anishnya — APPROVED** [review]
LGTM!

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,17 @@
 Changelog
 =========
 
+1.2.2 (2022-01-19)
+------------------
+
+- [NEW] Added Kazakh locale.
+- [FIX] The Czech, Slavic and Slovak locales now support ``dehumanize``.
```
Comment (anishnya) [inline CHANGELOG.rst:8]: For Slavic locales, do we want to be more specific?  I.E Russian, Polish, etc. ?

Comment (systemcatch) [reply CHANGELOG.rst:8]: Probably a good idea.

**Review by jadchaar — APPROVED** [review]
LGTM

**Review by krisfremen — APPROVED** [review]
LGTM

**Pull request merged**